core/templates/dev/head/domain/sidebar/sidebar-status.service.ts

```diff
@@ -17,9 +17,9 @@
  * hamburger-menu sidebar.
  */
 
+require('services/contextual/WindowDimensionsService.ts');
```

Comment:
Why was this added?

Reply:
This has not been added by me. I have not worked in the directory `services`, only in `domain`.

Reply:
The diff view doesn't lie :smile:, you have probably added it when you were resolving the merge conflicts.

Remove it, please.

Reply:
Done